charts/etcd/templates/etcd-statefulset.yaml

```diff
@@ -91,11 +91,16 @@ spec:
           protocol: TCP
         resources:
 {{ toYaml .Values.etcd.resources | indent 10 }}
+        env:
+        - name: ENABLE_TLS
+          value: {{ .Values.etcd.enableTLS }}
+        - name: BACKUP_ENDPOINT
+          value: "http{{ if .Values.etcd.enableTLS }}s{{ end }}://{{ .Values.name }}-local:{{ .Values.backup.port }}"
+        - name: FAIL_BELOW_REVISION_PARAMETER
+          value: "{{ if .Values.backup.failBelowRevision }}&failbelowrevision={{ int $.Values.backup.failBelowRevision }}{{ end }}"
```

Comment:
Just for my information, what are these for?

Reply:
Since bootstrap script will now reside on etcd custom image, we need a way to pass some "configurable" parts of the bootstrap script to the etcd container, like backup endpoint and whether tls is enabled or not (to set the root CA certs). Hence passing them as env vars. They can also be mounted as configmap values, but env vars seemed like the simpler option. You can check the current (old) bootstrap script to see all the config that's coming in from the chart's `values.yaml` file